### ISSUE TITLE
Inference does not give proper output.

### ISSUE THREAD

**Issue by harshhp8** [MathewAlexander/T5_nlg#2]
when I tried to test the model, it doesn't give output as mentioned. Have I done any mistakes,I can not figure it out.
@MathewAlexander you can look into this.

![t5-issue](https://user-images.githubusercontent.com/8132743/124718296-95806080-df23-11eb-8f7b-c12f2f32a677.png)

**Comment by MathewAlexander** [comment]
Did you execute the code in the proper order? did you load the model again from hugging face repo after training the model(This would override the variable "model")

**Comment by harshhp8** [comment]
In your repo, after the training phase, no code for downloading the hugging face model. Do you have that code?

**Comment by MathewAlexander** [comment]
It's already there on the code.
The steps are
1.Download the model
2. Fine-tune the model
3. Test the model
My question is, did you accidentally execute step 1 again after step 2

**Comment by MathewAlexander** [comment]
Also, Please send the screenshot of the training cell, would like to see the loss over epochs

**Comment by harshhp8** [comment]
> Also, Please send the screenshot of the training cell, would like to see the loss over epochs

Something is wrong.i didn't change any code. Attached screenshot.
![t5-issue_1](https://user-images.githubusercontent.com/8132743/124729604-36741900-df2e-11eb-9a89-9b76c2f64a27.png)
![t5-issue_2](https://user-images.githubusercontent.com/8132743/124729610-370caf80-df2e-11eb-8b2e-e36d8c717de6.png)

**Comment by MathewAlexander** [comment]
Try to run this [colab notebook](https://colab.research.google.com/drive/1bNocmqOWIXTCVTWrMxugUIvPTsfipDGS?usp=sharing) if you have not figured it out already

**Comment by harshhp8** [comment]
@MathewAlexander colab notebook is running fine. It was an indentation issue. Many thanks for your support.

**Comment by MathewAlexander** [comment]
@harshhp8  You are welcome!
